application/common/config/main.php

```diff
@@ -32,18 +32,14 @@
     $emailServiceConfig['validIpRanges'] = '';
 }
 
-if (empty($hrNotifierEmailTo)) {
-    $notifierConfig = ['class' => NullNotifier::class];
-} else {
```

Comment:
This change might cause a problem, as I think it would require all ID Sync instances to now provide the required configs for instantiating an EmailServiceNotifier, and any that aren't currently using one probably aren't yet providing those configs.

Reply:
They all have `emailServiceConfig` for log alerts. The only other config some are missing would be the `hrNotifierEmailTo` itself.

Reply:
And that's addressed by an `if` check in `sendMissingEmailNotice`. @mtompset had an issue with that solution, so I filed an Issue: https://github.com/silinternational/idp-id-sync/issues/123